pipeline/unified/channel/sign-release-package.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+parameters:
+    filePattern: null
+    inlineSignParams: null
+    signedArtifactName: null
+    platform: null
+
+steps:
+    - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@1
+      displayName: 'ESRP signing installer'
+      inputs:
+          ConnectedServiceName: 'ESRP Code Signing'
+          FolderPath: '$(System.DefaultWorkingDirectory)/signing-in-progress/${{ parameters.signedArtifactName }}/packed'
+          Pattern: ${{ parameters.filePattern }}
+          signConfigType: inlineSignParams
+          inlineOperation: ${{ parameters.inlineSignParams }}
+
+    - task: securedevelopmentteam.vss-secure-development-tools.build-task-codesignvalidation.CodeSign@1
```

Comment:
Since this is Windows only, should we just keep it in sign-release-package-windows.yaml? It would have to come after the publishing step, but that doesn't seem like a big issue--having the artifact might even help with debugging

Reply:
I put it here to ensure that the automatic canary release would be blocked if unsigned files are found. Now that I think about it, maybe the release would still be blocked either way if the task fails?